lib/merch_calendar/fiscal_year_calendar.rb

```diff
@@ -2,117 +2,89 @@
-    #         August 2016 = Offset retail month 1, year 2017 (2 quarters earlier)
-    def initialize(fy_quarter_offset = STITCH_FIX_FY_QUARTER_OFFSET)
-      @fy_quarter_offset = fy_quarter_offset
-
-      # TODO: support other fiscal year offsets
-      if fy_quarter_offset != STITCH_FIX_FY_QUARTER_OFFSET
-        raise NotImplementedError.new("FY quarter offset of #{fy_quarter_offset} not yet supported")
-      end
-
-      @retail_calendar = RetailCalendar.new
-    end
-
-    # The date of the first day of the year
     def start_of_year(year)
-      start_of_quarter(year, QUARTER_1)
+      end_of_year(year - 1) + 1
     end
-
-    # The date of the last day of the year
-    def end_of_year(year)
-      end_of_quarter(year, QUARTER_4)
-    end
-
-    # Return the starting date for a particular quarter
+    
     def start_of_quarter(year, quarter)
-      @retail_calendar.start_of_quarter(*offset_quarter(year, quarter))
+      case quarter
+      when QUARTER_1
+        start_of_month(year, 1)
+      when QUARTER_2
+        start_of_month(year, 4)
+      when QUARTER_3
+        start_of_month(year, 7)
+      when QUARTER_4
+        start_of_month(year, 10)
+      end
     end
-
-    # Return the ending date for a particular quarter
+    
     def end_of_quarter(year, quarter)
-      @retail_calendar.end_of_quarter(*offset_quarter(year, quarter))
-    end
-
-    # The date of the first day of the merch month
-    # @param [Fixnum] year - the fiscal year
-    # @param [Fixnum] merch_month - the nth month of the offset calendar
-    #   ex) for an offset of +/- 2 quarters, month 1 = August
-    def start_of_month(year, merch_month)
-      @retail_calendar.start_of_month(*offset_month(year, merch_month))
-    end
-
-    # The date of the last day of the merch month
-    # @param [Fixnum] year - the fiscal year
-    # @param [Fixnum] merch_month - the nth month of the offset calendar
-    #   ex) for an offset of +/- 2 quarters, month 1 = August
-    def end_of_month(year, merch_month)
-      @retail_calendar.end_of_month(*offset_month(year, merch_month))
-    end
-
-    # Returns the date that corresponds to the first day in the merch week
-    def start_of_week(year, merch_month, merch_week)
-      @retail_calendar.start_of_week(*offset_month(year, merch_month), merch_week)
-    end
-
-    # Returns the date that corresponds to the last day in the merch week
-    def end_of_week(year, merch_month, merch_week)
-      @retail_calendar.end_of_week(*offset_month(year, merch_month), merch_week)
-    end
-
-    # Returns the number of weeks in the fiscal year
-    def weeks_in_year(year)
-      @retail_calendar.weeks_in_year(offset_year(year))
+      case quarter
+      when QUARTER_1
+        end_of_month(year, 3)
+      when QUARTER_2
+        end_of_month(year, 6)
+      when QUARTER_3
+        end_of_month(year, 9)
+      when QUARTER_4
+        end_of_month(year, 12)
+      end
     end
 
-    private
+    def end_of_year(year)
```

Comment:
Reordering methods, and changing their behavior, makes it really hard on a reviewer to see what changed.  Can you move them back to their original ordering?

If you want to rearrange, I'd suggest doing it in another PR.